lstchain/scripts/lstchain_longterm_dl1_check.py

```diff
@@ -143,7 +147,11 @@ def main():
                   'min_altitude': [],
                   'mean_altitude': [],
                   'max_altitude': [],
-                  # currently (as of lstchain 0.5.3) event numbers are post-cleaning!:
+                  'min_azimuth': [],
+                  'max_azimuth': [],
+                  'mean_azimuth': [],
```

Comment:
Maybe nothing to change now, but this approach of dict of lists seems rather clumsy as you have multiple locations where you have to update if you add a new column.

I think a list of dicts would be much easier to maintain:

```
subrunwise_data = []

for subrun in subruns:
     data = {}

     data['charge_mean'] = ....
     

     subrunwise_data.append(data)

```


Like this, you only have a single place and much less to setup and pandas and astropy tables understand the format the same (its the difference between `DataFrame.to_dict()` and `DataFrame.to_dict(orient='records')` in pandas.